src/types.ts

```diff
@@ -13,3 +13,12 @@ export enum TagAttributes {
 export type Entity = Record<string, unknown>;
```

Comment:
We can use this one instead of all the `Record<string, unknown>` ;)
and maybe already merge it together with the SysProps?